libpod/networking_linux.go

```diff
@@ -162,9 +162,9 @@ func (r *Runtime) setupRootlessNetNS(ctr *Container) (err error) {
 	var cmd *exec.Cmd
 	if havePortMapping {
 		// if we need ports to be mapped from the host, create a API socket to use for communicating with slirp4netns.
-		cmd = exec.Command(path, "-c", "-e", "3", "-r", "4", "--api-socket", apiSocket, fmt.Sprintf("%d", ctr.state.PID), "tap0")
+		cmd = exec.Command(path, "--mtu=65520", "-c", "-e", "3", "-r", "4", "--api-socket", apiSocket, fmt.Sprintf("%d", ctr.state.PID), "tap0")
```

Comment:
Is this valid? Or should this be 
```
cmd = exec.Command(path, "--mtu", "65520", "-c", ...
```

Reply:
slirp4netns accepts `--mtu=VALUE`, I've tried it manually to be sure:

```
$ ./slirp4netns --mtu=1234 25691 foo
sent tapfd=5 for foo
received tapfd=5
Starting slirp
* MTU:             1234
* Network:         10.0.2.0
* Netmask:         255.255.255.0
* Gateway:         10.0.2.2
* DNS:             10.0.2.3
* Recommended IP:  10.0.2.100

```

Reply:
But the shell might clean this up for you.  
Or does slirp4netns not work with
slirp4netns --mtu 1234 25691 foo

Reply:
slirp4netns works with that as well.  I've pushed a new version